src/text-input/index.ts

```diff
@@ -33,6 +33,8 @@ export interface TextInputProperties extends ThemedProperties, InputProperties,
 	placeholder?: string;
 	value?: string;
 	shouldFocus?: boolean;
+	pattern?: string | RegExp;
+	autocomplete?: string;
```

Comment:
we should make this a `boolean` and then reflect the right value depending on `true` or false (which would mean it needs to be in `properties` list for the custom element def too.

Reply:
`autocomplete` can be more than `"on"` and `"off"`: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete

Reply:
Urgh

Reply:
`boolean | string`?